mitxgraders/baseclasses.py

```diff
@@ -207,39 +207,76 @@ def log_output(self):
         content = "\n".join(self.debuglog)
         return "<pre>{content}</pre>".format(content=content)
 
-    @classmethod
-    def ensure_text_inputs(cls, student_input, allow_lists=True, allow_single=True):
+    @staticmethod
+    def ensure_list_of_text_inputs(student_input):
         """
-        Ensures that student_input is a list of text strings.
+        Delegated to by AbstractGrader.ensure_text_inputs. Validates whether
+        student_input is a list of text strings.
 
-        NOTE: Overriden by ItemGrader to ensure input is a single text string.
+        Returns:
+        A pair (result, errmsg) where either:
+            - result is a list of text strings and errmsg is None, or
```

Comment:
A pair??? That... doesn't make sense. It returns either the result, or raises an error message if something went wrong...